l10n_ch_invoice_with_payment/models/report.py

```diff
@@ -9,12 +9,17 @@ class IrActionsReport(models.Model):
 
     @api.multi
     def render_qweb_pdf(self, res_ids=None, data=None):
+        ctx = self.env.context
+        # Remove context key that comes from invoice action
+        if ctx.get('default_type'):
+            ctx = ctx.copy()
+            ctx.pop('default_type')
```

Comment:
maybe this should go after line 19?

Reply:
This context key comes from `account.invoice` the current object is already an `ir.actions.report` to me it shouldn't be there at all.